style.css

```
@@ -1,11 +1,224 @@
-/* CSS files add styling rules to your content */
+/* ====================
+    GLOBAL STUFF
+   ====================*/
+
+* {
+  box-sizing: border-box;
+}
 
 body {
-  font-family: helvetica, arial, sans-serif;
+  background-color: #fdf6dd;
+  font-family: "Source Sans Pro", arial, sans-serif;
   margin: 2em;
 }
 
+.container {
+  color: #000000;
+  margin: 0 auto;
+  padding: 0;
+  position: relative;
+  width: 800px;
+}
+
+.container div {
+  background-color: #fff;
+  border-radius: 15px;
+  margin: 15px;
+  padding: 20px;
+}
+
 h1 {
-  font-style: italic;
-  color: #373fff;
+  font-size: 40px;
+  margin: 0;
+  padding: 10px 0;
+}
+
+h2 {
+  font-size: 36px;
+  margin: 0 0 10px 0;
+  padding: 10px;
+}
+
+h3 {
+  font-size: 30px;
+  margin: 0 0 10px 0;
+  padding: 10px;
+}
+
+ul {
+  margin: 0 45px;
+  padding: 0;
+}
+
+h4 {
+  margin: 0;
+  padding: 5px;
+}
+
+p {
+  color: black;
+  font-size: 14px;
+  margin: 2px;
+  padding: 5px;
+  margin-left: 20px;
+}
+
+li p {
+  margin: 0 0 0 5px;
+}
+
+
+/*  ===================== 
+    LAYOUT
+    ===================== */
+
+/*  ------ HEADER ------ */
+
+.p-info {
+  height: 400px;
+  position: relative;
+}
+
+.p-info h1 {
+  left: 60px;
+  margin: auto 0;
+  position: relative;
+  top: 180px;
+  width: 225px;
+  z-index: 1;
+}
+
+.p-info p {
+  left: 60px;
+  margin-left: 15px;
+  padding: 0;
+  position: relative;
+  text-align: left;
+  top: 180px;
+  width: 225px;
+  z-index: 1;
+}
+
+.portrait {
+  border: 15px #fff solid;
+  border-radius: 175px;
```

This feels like a _very_ specific number; for creating circles, consider using:
```suggestion
  border-radius: 50%;
```